datafusion/core/tests/fuzz_cases/window_fuzz.rs

```diff
@@ -415,36 +414,6 @@ fn get_random_function(
             ),
         );
     }
-    window_fn_map.insert(
-        "first_value",
-        (
-            WindowFunctionDefinition::BuiltInWindowFunction(
-                BuiltInWindowFunction::FirstValue,
-            ),
-            vec![arg.clone()],
-        ),
-    );
-    window_fn_map.insert(
-        "last_value",
-        (
-            WindowFunctionDefinition::BuiltInWindowFunction(
-                BuiltInWindowFunction::LastValue,
-            ),
-            vec![arg.clone()],
-        ),
-    );
-    window_fn_map.insert(
-        "nth_value",
-        (
-            WindowFunctionDefinition::BuiltInWindowFunction(
-                BuiltInWindowFunction::NthValue,
-            ),
-            vec![
-                arg.clone(),
-                lit(ScalarValue::Int64(Some(rng.gen_range(1..10)))),
-            ],
-        ),
-    );
```

Comment:
Same applies here. Fix the test and not delete it.

Reply:
done thanks